src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceTransformer.java

```diff
@@ -676,6 +675,9 @@ private void addSettingsImports(InterfaceContext context) {
     typeTable.saveNicknameFor("javax.annotation.Generated");
     typeTable.saveNicknameFor("org.threeten.bp.Duration");
 
+    // TODO(andrealin): remove this from httpjson
+    typeTable.saveNicknameFor("com.google.api.gax.grpc.GrpcExtraHeaderData");
```

Comment:
I don't quite understand this TODO - this class isn't even accessible from httpjson.

Reply:
Ok, this import has been applied only to proto-input surfaces.